Watching for file changes with StatReloader
Not Found: /
Internal Server Error: /clients/
Not Found: /clients/1/
Watching for file changes with StatReloader
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
D:\FORMATION_PYTHON\PROJETS\P12\crm\crm\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /
Not Found: /favicon.ico
Not Found: /users/
Internal Server Error: /clients/
Internal Server Error: /clients/
Internal Server Error: /clients/
Traceback (most recent call last):
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\rest_framework\mixins.py", line 38, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "D:\FORMATION_PYTHON\PROJETS\P12\crm\create_events\views.py", line 44, in get_queryset
    event = Event.objects.get(support_contact=user)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "D:\FORMATION_PYTHON\PROJETS\P12\venv\lib\site-packages\django\db\models\query.py", line 496, in get
    raise self.model.DoesNotExist(
create_events.models.Event.DoesNotExist: Event matching query does not exist.
Unauthorized: /userlist/
Forbidden: /userlist/
